scripts/wca-bootstrap.sh

```diff
@@ -74,12 +76,7 @@ if [ "$environment" != "development" ]; then
   # You'll need ssh access to worldcubeassociation.org as user `cubing`. Contact
   # software-admins@worldcubeassociation.org if you need access.
   echo "Downloading secrets from worldcubeassociation.org..."
-  rsync -az -e "ssh -o StrictHostKeyChecking=no" --info=progress2 cubing@worldcubeassociation.org:/home/cubing/worldcubeassociation.org/secrets/ $repo_root/secrets
-
-  if [ "$environment" == "staging" ]; then
-    echo "Downloading certificate from staging.worldcubeassociation.org..."
-    rsync -az -e "ssh -o StrictHostKeyChecking=no" --info=progress2 cubing@staging.worldcubeassociation.org:/home/cubing/worldcubeassociation.org/secrets/https/ $repo_root/secrets/https
-  fi
+rsync -az -e "ssh -o StrictHostKeyChecking=no" --info=progress2 cubing@$PRODUCTION_ELASTIC_IP:/home/cubing/worldcubeassociation.org/secrets/ $repo_root/secrets
```

Comment:
Same as above: Indentation seems off, though this could just be an issue in the GitHub diff viewer.